_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| ----------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {% include anchor a="error.transport.challenges.challengeTypeRequiresRelationship" %}                             | The challenge type 'Relationship' requires a Relationship for the challenge creation.                                                                                                                                                                           |
+| {% include anchor a="error.transport.datawallet.currentBiggerThanTarget" %}                                       | The current datawallet version is bigger than the target version.                                                                                                                                                                                               |
+| {% include anchor a="error.transport.datawallet.encryptedPayloadIsNoCipher" %}                                    | The given encrypted payload is no cipher.                                                                                                                                                                                                                       |
+| {% include anchor a="error.transport.datawallet.insufficientSupportedDatawalletVersion" %}                        | The current SupportedDatawalletVersion is too low to perform the requested operation. This means that the datawallet of the identity was upgraded by another device with a higher version. It is necessary to update the current device.                        |
+| {% include anchor a="error.transport.datawallet.unsupportedModification" %}                                       | A collection was recieved in a CacheChanged datawallet modification but is not supported in the current version of the library.                                                                                                                                 |
+| {% include anchor a="error.transport.devices.alreadyOnboarded" %}                                                 | The device that you try to create an onboarding code for has already been onboarded.                                                                                                                                                                            |
```

Comment:
If I am not mistaken, we create a QR code to onboard a device without knowing which device will use it for onboarding. In this case, I don't understand the description.

Reply:
Yes this was described false before, I changed it now.